cylc/uiserver/app.py

```diff
@@ -81,21 +79,17 @@
     UIServerGraphQLHandler,
     UserProfileHandler,
 )
+from cylc.uiserver.jupyterhub_config import (
+    get_conf_dir_hierarchy,
+    SITE_CONF_ROOT,
+    USER_CONF_ROOT
+)
```

Comment:
Because of the way JupyterHub config files work the `jupyterhub_config` file auto loads so we should not import it.

Can move the function and vars into this file or put them in another module.